src/SeleniumLibrary/keywords/element.py

```diff
@@ -511,6 +511,13 @@ def focus(self, locator):
         """Deprecated. Use `Set Focus To Element` instead."""
         self.set_focus_to_element(locator)
 
+    @keyword
+    def scroll_element_into_view(self, locator):
+        """Scrolls an element from given ``locator`` into view.
+        """
+        element = self.find_element(locator)
+        self.driver.execute_script("arguments[0].scrollIntoView()", element)
```

Comment:
I know that some users are actually using the library on places where JavaScript is not available. Therefore it should be better use Selenium ActionChains. Example like this: `ActionChains(self.driver).(element).perform()`

Reply:
I see. However, ActionChains has no method to do that.
https://seleniumhq.github.io/selenium/docs/api/py/webdriver/selenium.webdriver.common.action_chains.html.

Reply:
There is this one: https://seleniumhq.github.io/selenium/docs/api/py/webdriver/selenium.webdriver.common.action_chains.html#selenium.webdriver.common.action_chains.ActionChains.move_to_element

Reply:
This only hovers element. I tested in Firefox and it does not scroll.

Reply:
It works with Chrome and Edge. Also there is an exception raised when trying with Firefox. 
```
Traceback (most recent call last):
  File "test.py", line 16, in <module>
    ActionChains(driver).move_to_element(element).perform()
  File "/usr/local/lib/python2.7/dist-packages/selenium/webdriver/common/action_chains.py", line 80, in perform
    self.w3c_actions.perform()
  File "/usr/local/lib/python2.7/dist-packages/selenium/webdriver/common/actions/action_builder.py", line 76, in perform
    self.driver.execute(Command.W3C_ACTIONS, enc)
  File "/usr/local/lib/python2.7/dist-packages/selenium/webdriver/remote/webdriver.py", line 312, in execute
    self.error_handler.check_response(response)
  File "/usr/local/lib/python2.7/dist-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.MoveTargetOutOfBoundsException: Message: (183.5, 5330.85009765625) is out of bounds of viewport width (1280) and height (779)
```
I did raise an issue to the Selenium issue tracker: https://github.com/SeleniumHQ/selenium/issues/5601. For mean time, make: if browser is Firefox then use JavaScript, else use `ActionChains`

Reply:
Does it scroll element on my test scroll page?

Reply:
I didn't try your test page, but this did work with Chrome and Edge: https://gist.github.com/aaltat/a7f71e960d1bc3c6f90bfebb216162f0

Reply:
I tested it in Chrome and action chain solution works.